### ISSUE TITLE
Switch color gradient to color vision deficiency friendly version.

### ISSUE THREAD

**Issue by blowerd** [blowerd/tug-hill-grantmaker#3]
Current method uses reds and blues for the purposes of illustration. In line with accessibility requirements, this should be migrated to a more universally perceivable gradient.
